frontend/src/GameReports.tsx

```diff
@@ -0,0 +1,207 @@
+import axios from "axios";
+import React, { useEffect, useState } from "react";
+import {
+    Competition,
+    Expansion,
+    League,
+    Match,
+    ProcessedGameReport,
+    Side,
+    Stronghold,
+    Victory,
+} from "./types";
+import {
+    getExpansionLabel,
+    getLeagueLabel,
+    getStrongholdLabel,
+    strongholdPoints,
+    strongholdSide,
+} from "./utils";
+import TableView from "./TableView";
+
+export default function GameReports() {
+    const [reports, setReports] = useState<ProcessedGameReport[]>([]);
+    const [loading, setLoading] = useState(false);
+    const [error, setError] = useState<string | null>(null);
+
+    const getReports = async () => {
+        try {
+            const response = await axios.get(
+                // "http://localhost:8081/leaderboard"
+                "https://api.waroftheringcommunity.net:8080/reports"
+            );
+            setReports(response.data.reports);
+        } catch (error) {
+            setError("Something went wrong");
+            console.error(error);
+        }
+        setLoading(false);
+    };
+
+    const refresh = () => {
+        setError(null);
+        setLoading(true);
+        getReports();
+    };
+
+    useEffect(refresh, []);
```

Comment:
Planning to move all this into a hook so it can be reused between `GameReports` and `Rankings`, but for now it's just copy pasted